### PR TITLE
Add document for SUSE Private Registry

### PR DESCRIPTION
**Pull request by r0ckarong**

### CODE REVIEW

**Comment by r0ckarong** [comment]
@chabowski Now waiting for reviews by Ecosystem team. Please do not merge yet.

**Comment by chabowski** [comment]
> @chabowski Now waiting for reviews by Ecosystem team. Please do not merge yet.

@r0ckarong of course not ;-) - waiting for your go. Also - Travis has not yet finalized the checks - if something fails, we should fix that before we merge.

File: adoc/Harbor-Private-Registry-Scenarios.adoc
Comment (jsuchome) [inline adoc/Harbor-Private-Registry-Scenarios.adoc:3]: I think this line should be right before the table, i.e. after the bash code block.

File: adoc/Harbor-Private-Registry-Installation.adoc
Comment (jsuchome) [inline adoc/Harbor-Private-Registry-Installation.adoc:685]: Well, this is bit confusing. Managed Redis can be used together with on-premise kubernetes cluster as well, not just when deployed in AKS or EKS.
I think 

_As an alternative to using the Redis Operator, {spr} may instead be connected to a managed Redis instance running in public cloud._

alone is correct

Comment (jsuchome) [reply adoc/Harbor-Private-Registry-Installation.adoc:685]: cc @stefannica @bbobrov

File: adoc/Harbor-Private-Registry-Installation.adoc
Comment (jsuchome) [inline adoc/Harbor-Private-Registry-Installation.adoc:699]: This line above ("addr for redis:... ") is actually a comment. Either remove it or mark it correctly.

File: adoc/Harbor-Private-Registry-Installation.adoc
Comment (jsuchome) [inline adoc/Harbor-Private-Registry-Installation.adoc:713]: Same as above, this line is comment.

File: adoc/Harbor-Private-Registry-Installation.adoc
Comment (jsuchome) [inline adoc/Harbor-Private-Registry-Installation.adoc:715]: There should be a note here that the password is the value used when preparing Redis in AWS...

File: adoc/Harbor-Private-Registry-Installation.adoc
Comment (jsuchome) [inline adoc/Harbor-Private-Registry-Installation.adoc:718]: Looks like a newline is missing here? The text is not shown correctly as a caption ...

File: adoc/Harbor-Private-Registry-Installation.adoc
Comment (jsuchome) [inline adoc/Harbor-Private-Registry-Installation.adoc:774]: I think _<ingress_url>_ was actually replaced by _<core_fqdn>_ at the start of installation section...

File: adoc/Harbor-Private-Registry-Installation.adoc
Comment (r0ckarong) [reply adoc/Harbor-Private-Registry-Installation.adoc:715]: That is not something we described in this document, right?

File: adoc/Harbor-Private-Registry-Troubleshooting.adoc
Comment (jsuchome) [inline adoc/Harbor-Private-Registry-Troubleshooting.adoc:24]: The error line (_FailedScheduling <unknown> default-scheduler pod has unbound immediate `PersistentVolumeClaims` (repeated 2 times)?_) should be formatted separately from the paragraph text...

File: adoc/Harbor-Private-Registry-Administration.adoc
Comment (jsuchome) [inline adoc/Harbor-Private-Registry-Administration.adoc:8]: <ingress_url> was actually replaced by <core_fqdn> at the start of installation section...

File: adoc/Harbor-Private-Registry-Administration.adoc
Comment (jsuchome) [inline adoc/Harbor-Private-Registry-Administration.adoc:9]: <<install-passwords> is not leading anywhere. Is this coming from the original confluence content? Maybe we need some TODO here

File: adoc/Harbor-Private-Registry-Installation.adoc
Comment (r0ckarong) [reply adoc/Harbor-Private-Registry-Installation.adoc:718]: The anchors inside the ordered lists are a major headache. I will change to inline anchor and see if that works.

File: adoc/Harbor-Private-Registry-Administration.adoc
Comment (jsuchome) [inline adoc/Harbor-Private-Registry-Administration.adoc:13]: Too many _admin_s here. As you can see in the picture, there's only one menu item call _admin_ and then there's Change Password.

File: adoc/Harbor-Private-Registry-Installation.adoc
Comment (jsuchome) [reply adoc/Harbor-Private-Registry-Installation.adoc:715]: Yeah, that's part of (still missing) _Connect to an Amazon ElastiCache Redis service_ section. I think you can actually create such (empty) section after _Azure Cache for Redis_ one.

File: adoc/Harbor-Private-Registry-Userguide.adoc
Comment (jsuchome) [inline adoc/Harbor-Private-Registry-Userguide.adoc:41]: s/2.0.0/2.1.0

File: adoc/Harbor-Private-Registry-Userguide.adoc
Comment (jsuchome) [inline adoc/Harbor-Private-Registry-Userguide.adoc:86]: Replace 2.0.0 with 2.1.0

Comment (r0ckarong) [reply adoc/Harbor-Private-Registry-Userguide.adoc:86]: Did a replace across the docs for this

File: adoc/Harbor-Private-Registry-Userguide.adoc
Comment (jsuchome) [inline adoc/Harbor-Private-Registry-Userguide.adoc:102]: _kubernetes_node_ (replace the dot)

File: adoc/Harbor-Private-Registry-Userguide.adoc
Comment (jsuchome) [inline adoc/Harbor-Private-Registry-Userguide.adoc:172]: 2.1.0 again

File: adoc/Harbor-Private-Registry-Userguide.adoc
Comment (jsuchome) [inline adoc/Harbor-Private-Registry-Userguide.adoc:188]: I think we have `notary_fqdn` in the installation_steps section

File: adoc/Harbor-Private-Registry-Userguide.adoc
Comment (jsuchome) [inline adoc/Harbor-Private-Registry-Userguide.adoc:226]: Exactly: I do not know. So unless we find out, we better remove that sentence.

(I have a feeling that it can be done with `notary` command line client, which we do **not** ship.)

**Comment by dirkmueller** [comment]
are we going to see the filenames anywhere? "Harbor-private-registry-*" is a bit controversial, either suse-private-registry or private-registry* would be less controversial.

**Comment by r0ckarong** [comment]
> are we going to see the filenames anywhere? "Harbor-private-registry-_" is a bit controversial, either suse-private-registry or private-registry_ would be less controversial.

Unless you visit the Github repository they won't show up but the repository is public. I can just change the names.

File: adoc/Private-Registry-docinfo.xml
Comment (dirkmueller) [inline adoc/Private-Registry-docinfo.xml:10]: whats this?

**Review by dirkmueller — APPROVED** [review]
LGTM for a conversion. Thanks Markus. lets move this forward so that we can iterate.

File: adoc/Private-Registry-docinfo.xml
Comment (r0ckarong) [reply adoc/Private-Registry-docinfo.xml:10]: Just some leftover artifacts from the file I used as a template for this metadata info. I'll remove the comments.

**Pull request merged**
